proxy/certs/certs.go

```diff
@@ -226,6 +230,13 @@ func (s *RemoteCertSource) Local(instance string) (tls.Certificate, error) {
 	if err != nil {
 		return tls.Certificate{}, fmt.Errorf("couldn't parse ephemeral certificate for instance %q: %v", instance, err)
 	}
+	if s.EnableIAMLogin {
+		// Adjust the certificate's expiration to be the earlier of c.NotAfter
+		// or tok.Expiry.
```

Comment:
nit: Is this line break really worth it for this comment?

Reply:
I'm a sucker for 80 characters, but will remove the second line here.

Reply:
How very un go-like 😉 

If we were keeping consistent with 80 chars I wouldn't say anything but since we aren't it doesn't make much sense.